homeassistant/components/wallbox/config_flow.py

```diff
@@ -0,0 +1,114 @@
+"""Config flow for Wallbox integration."""
+import logging
+
+import requests
+import voluptuous as vol
+from wallbox import Wallbox
+
+from homeassistant import config_entries, core, exceptions
+
+from .const import DOMAIN
+
+COMPONENT_DOMAIN = DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        "station": str,
```

Comment:
Please make the keys constants and refer to e.g. STATION and not "station".

Reply:
OK, changed this to use constants from .const and homeassistant.const

Reply:
I cannot see it.

Reply:
no, did this locally, will make a big commit when I have processed more changes

Reply:
ok, and seen from that POW it is totally correct not to resolve the conversation